### PR TITLE
Added ClueLess to Hacktoberfest Swag List

### PR DESCRIPTION
**Pull request by trishitapingolia**
#615

Thanks for contributing to the [Hacktoberfest Swag List](https://hacktoberfestswaglist.com/) :smiley: :tada:! Before submitting your pull request, please check off as many of the items below as you can:

1. [x] I have read the [Contributing.md](https://github.com/crweiner/hacktoberfest-swag-list/blob/master/docs/contributing.md) file and formatted this PR correctly
2. [x] I'm not adding a company from the blocklist
3. [x] I make sure to fix things promptly if an error or suggestion comes up

Thanks and Happy Hacktoberfest! :tada:
Tagging @crweiner to take a look. :eyes:

### CODE REVIEW

File: docs/list.md [hunk 1]
```diff
@@ -134,6 +134,15 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
 - **How to sign up**: Once all 4 PRs are accepted, then [fill out the form here](https://camunda.com/hacktoberfest2022/){:target="\_blank"}.
 - **Notes**: Additional information regarding their Hacktoberfest participation [can be found on this page](https://camunda.com/hacktoberfest2022/){:target="\_blank"}.
 
+#### **ClueLess**
+
+- **Swag**: Certificate for Contributing, T-shirt
```
[the remark this answers is not in the document]
Comment (crweiner) [reply docs/list.md:139]: This is not yet resolved, please remove the whitespace after T-shirt. 👕

File: docs/list.md [hunk 2]
```diff
@@ -134,6 +134,15 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
 - **How to sign up**: Once all 4 PRs are accepted, then [fill out the form here](https://camunda.com/hacktoberfest2022/){:target="\_blank"}.
 - **Notes**: Additional information regarding their Hacktoberfest participation [can be found on this page](https://camunda.com/hacktoberfest2022/){:target="\_blank"}.
 
+#### **ClueLess**
+
+- **Swag**: Certificate for Contributing, T-shirt 
+- **Requirements**:
```
[the remark this answers is not in the document]
Comment (crweiner) [reply docs/list.md:140]: This is not yet resolved, please remove the whitespace.

File: docs/list.md [hunk 3]
```diff
@@ -134,6 +134,15 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
 - **How to sign up**: Once all 4 PRs are accepted, then [fill out the form here](https://camunda.com/hacktoberfest2022/){:target="\_blank"}.
 - **Notes**: Additional information regarding their Hacktoberfest participation [can be found on this page](https://camunda.com/hacktoberfest2022/){:target="\_blank"}.
 
+#### **ClueLess**
+
+- **Swag**: Certificate for Contributing, T-shirt 
+- **Requirements**: 
+    - All contributors will receive certificate for contributing.
+    - Top Contributor will get Exclusive t-shirts for free.
+- **How to sign up**: Check out the issues [here](https://github.com/Clueless-Community/first-contribution){:target="\_blank"}
```
Comment (crweiner) [inline docs/list.md:143]: `**How to sign up**` is for how this organization is tracking your contributions to see what swag you'll earn. Do you see how they're keeping track? Do you have to sign up for anything?

File: docs/list.md [hunk 4]
```diff
@@ -134,6 +134,15 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
...
+- **Notes**: For more info click [here](https://www.clueless.tech/hacktoberfest#)
```
Comment (crweiner) [inline docs/list.md:144]: Please double check that you add links based on the [Contribuing guide](https://hacktoberfestswaglist.com/contributing/#a-to-z-order-of-companies) 👍

**Review by crweiner — CHANGES_REQUESTED** [review]
Hello @thehungrycoder! 👋 Thank you for your PR to the Hacktoberfest Swag List! I've left some feedback below for you to address, please. If you could also add a little more detail overall and fix the merge conflict, I'd appreciate it! 👍 

Please let me know if you have any questions or would like me to take another look. 👀

**Comment by trishitapingolia** [comment]
> Hello @thehungrycoder! 👋 Thank you for your PR to the Hacktoberfest Swag List! I've left some feedback below for you to address, please. If you could also add a little more detail overall and fix the merge conflict, I'd appreciate it! 👍
> 
> Please let me know if you have any questions or would like me to take another look. 👀

Thanks for taking a look at it, I'll make the changes accordingly.

**Comment by crweiner** [comment]
Thank you for your PR, @trishitapingolia! 🙇 ClueLess has now been added in PR #652. There's lots more organization to add to the Swag List (see the Issues tab) if you'd like to grab another one! Thank you, I hope to see another PR from you soon, and have a great Hacktoberfest! 👍

**Comment by thehungrycoder** [comment]
> Hello @thehungrycoder! 👋 Thank you for your PR to the Hacktoberfest Swag List! I've left some feedback below for you to address, please. If you could also add a little more detail overall and fix the merge conflict, I'd appreciate it! 👍
> 
> Please let me know if you have any questions or would like me to take another look. 👀

I'm not following it well. I think my PR was last year! I'm not sure if IBM has similar offering this year. I'll have to check.

**Comment by crweiner** [comment]
Whoops!!! So sorry about that @thehungrycoder. I have no idea why that pulled you up to ping! My apologies. 😅

**Pull request closed without merging**